src/Microsoft.DotNet.Interactive.Formatting.Tests/FormatterTests.cs

```diff
@@ -644,21 +630,6 @@ public void Formatters_can_be_registered_on_demand_for_open_generic_interfaces(s
                 .Be(list.Count.ToString());
         }
 
-        [Fact]
-        public void When_JArray_is_formatted_it_outputs_its_string_representation()

```

Comment:
Is there an equivalent test we should add for System.Text.Json?

Reply:
we have those tests for the types in System.Text.Json